pkg/util/file_indexer.go

```diff
@@ -8,35 +8,59 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const fileIndexDirectory = ".odo"
 const fileIndexName = "odo-file-index.json"
 
+// FileIndex holds the file index used for storing local file state change
+type FileIndex struct {
```

Comment:
OdoFileIndex

Reply:
Why prepend it with `Odo`? Isn't it a little bit redundant?

Reply:
Yeah as its not going to be used by something outside odo, I do we need it?

Reply:
> Yeah as its not going to be used by something outside odo

Since the struct isn't useful outside odo, just wanted it in the prefix to indicate that it's for odo's internal use only

Reply:
with that logic wouldn't we prefix every single struct in our project with `Odo`?

Reply:
ok makes sense